src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState, useEffect } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { client } from '../../utils/fetchClient';
+import { ErrorMessage } from '../../types/ErrorStatusType';
+
+type TodoItemProps = {
+  todo: Todo;
+  todos?: Todo[] | null;
+  setTodos?: (todos: Todo[]) => void;
+  isTempTodo: boolean;
+  setErrorMessage?: (errorMessage: ErrorMessage) => void;
+  todoIdsToDelete?: number[];
+};
+
+export const TodoItem: React.FC<TodoItemProps> = ({
+  todo,
+  todos,
+  isTempTodo,
+  setTodos,
+  setErrorMessage,
+  todoIdsToDelete,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    if (todoIdsToDelete && todoIdsToDelete.includes(todo.id)) {
+      setIsLoading(true);
+    } else {
+      setIsLoading(false);
+    }
+  }, [todoIdsToDelete, todo.id]);
```

Comment:
```suggestion
  const isLoading = todoIdsToDelete && todoIdsToDelete.includes(todo.id);
```
You don't need a state, variable is enough

Reply:
Not fixed, please, do these changes in the next task

Reply:
Sorry, forgot to mention - I changed the state to variable in the third part.